resync/api/chat.py

```
@@ -315,11 +358,33 @@ async def websocket_endpoint(
             await websocket.close(code=1008, reason="Authentication required")
             return
     except Exception:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)